src/pages/team.js

```diff
@@ -71,37 +70,49 @@ class Team extends Component {
                         />
                         {' '}Team
                     </Header>
-                    <Container>
-                        <Header as="h2" className="officers-header">{ this.state.activeTeam === "Leadership" ? "Executive Board" : "Team Leads" }</Header>
-                        <div className="officers-container">
-                            {
-                                officers.filter(officer => {
-                                    if (this.state.activeTeam === "Leadership") {
-                                        const reducer = (acc, curr) => officer.position.includes(curr) || acc
-                                        return executiveBoard.reduce(reducer, false)
-                                    } else {
-                                        return officer.position.includes(this.state.activeTeam + " Chair")
+                    {
+                        this.state.activeTeam !== "Alumni" ?
+                        <>
+                            <Container>
```

Comment:
gonna make a note to probably refactor the alumni/regular team display into two stateless components but i won't torture you further on this pr :P